README.md

```diff
@@ -1,2 +1,34 @@
 # overops-functions
 Public OverOps User Defined Functions
+
+Full manifest of all libraries can be found here - https://git.io/fxDIW
+
+## Our Collection consists of two types of functions:
```

Comment:
perhaps we should also mention the new "install" function that runs when the UDF is saved